src/comment.rs

```diff
@@ -1703,12 +1703,11 @@ impl<'a> Iterator for CommentCodeSlices<'a> {
 }
 
 /// Checks is `new` didn't miss any comment from `span`, if it removed any, return previous text
-/// (if it fits in the width/offset, else return `None`), else return `new`
```

Comment:
Before updating recover_comment_removed, it always returned `Some(String)`. The previous comment about this function is no longer accurate, as it was written when the function used to call `wrap_str` a long time ago.